eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1649,18 +1649,29 @@ public function deleteVersion(APIVersionInfo $versionInfo)
      * Loads all versions for the given content.
      *
      * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the user is not allowed to list versions
+     * @throws \eZ\Publish\Core\Base\Exceptions\InvalidArgumentException if invalid status
      *
      * @param \eZ\Publish\API\Repository\Values\Content\ContentInfo $contentInfo
+     * @param int $status
      *
      * @return \eZ\Publish\API\Repository\Values\Content\VersionInfo[] Sorted by creation date
      */
-    public function loadVersions(ContentInfo $contentInfo)
+    public function loadVersions(ContentInfo $contentInfo, $status = null)
```

Comment:
I'm wondering about introducing struct representing "loading options" here: In the feature, we might need to support other filters (e.g. creator) and pagination

Reply:
I agree. It would be nice to have something like that, but honestly, I really don't want to do anything more than what is necessary for improving the mentioned command. 
Good idea, but it should go to `master` then 😉

Reply:
And we don't currently do struct for loading options, we did for a bit on load content, but we dropped it when we found a way around.